src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -36,9 +39,10 @@
     private static final String REGEX_BLOB_KEY = "(encoded_gs_key:)?[a-zA-Z0-9-_]{10,}";
     private static final String REGEX_QUESTION_ID = "[a-zA-Z0-9-_]{40,}";
     private static final String REGEX_COMMENT_ID = "[0-9]{16}";
-    private static final String REGEX_DISPLAY_TIME = "(0[0-9]|1[0-2]):[0-5][0-9] [AP]M( UTC)?";
+    private static final String REGEX_DISPLAY_TIME = "(0[0-9]|1[0-2]):[0-5][0-9] [AP]M?";
     private static final String REGEX_ADMIN_INSTITUTE_FOOTER = ".*?";
     private static final String REGEX_SESSION_TOKEN = REGEX_UPPERCASE_HEXADECIMAL_CHAR_32;
+    private static final String REGEX_TIMEZONE_OFFSET = "(([\\+][0-9][0-9][0-9][0-9])|([\\-][0-9][0-9][0-9][0-9]))";
```

Comment:
The regex is long and quite hard to read. `"UTC([+-]\\d{4})"` could be one better way to write this.